README.md

```diff
@@ -1 +1,59 @@
-# Group Capstone Project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+#  JavaScript Capstone Project
+> New Movies website is a simple app that takes data from API and displays films on the user's screen, he can like, comment and reserve for a movie
+
+# Built With
+
+- Html
+- Css
+- Js
+- Git & Gitflow
+- webpack
+- TvMaze api 
+
+
+
+## Getting Started
+To run the above project you need to meet the following requirements:
+
+
+## Prerequisites
+- Have [Vscode](https://code.visualstudio.com/) installed 
+- Have [Git](https://git-scm.com/) installed
+- Node installed
+
+## Documentation 
+
+[Video link](https://drive.google.com/file/d/1pM362SpFZWo3NLE3K1kejYNdExx12B5L/view?usp=sharing)
+
```

Comment:
Good job guys on your project work. There are a few changes you have to work on the video so far.

- [ ] Kindly redo the video and have the length a maximum of 5 mins. It's required that the maximum length of the video is 5 minutes and the one you guys did goes up to 9 minutes. 

**HINT**
- Kindly be precise and make it short in around 5 minutes
- For some feature, you present, just do it once. For example, in doing a comment, one example is okay. 👍
- Also please don't forget to talk a brief about a section in your code you're most proud of implementing.
- Be summarize as much as you can within that 5 minutes 👍